### PR TITLE
28). Fix output message for all articles

### PR DESCRIPTION
**Pull request by jameshkramer**
This PR is for user story #1576037.

It changes the output message from "The example produces this output:" to "This example produces the following output:" in all prior articles, including some which have not yet been merged.  Please merge #17928, #18022, #18050, and #18081 before merging this.

cc: @BillWagner @mairaw @tfosmark @marteeleigh

Contributes to #4728

### CODE REVIEW

**Comment by mairaw** [comment]
Can you fix the merge conflicts here @jameshkramer? We should be able to merge this one independently.

File: docs/standard/linq/comparison-xpath-linq-xml.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+---
```
Comment (mairaw) [inline docs/standard/linq/comparison-xpath-linq-xml.md:1]: I didn't expect to see new articles on this PR @jameshkramer. By design?

Comment (jameshkramer) [reply docs/standard/linq/comparison-xpath-linq-xml.md:1]: That article is added in br26. I think we should merge br26 before br28.

Comment (mairaw) [reply docs/standard/linq/comparison-xpath-linq-xml.md:1]: Sounds good.

**Comment by jameshkramer** [comment]
@BillWagner @mairaw 

Markdown lint is reporting an error, and showing a lot of messages for files that are not ours. What's going on, and what do I do?

**Comment by BillWagner** [comment]
@jameshkramer 

> Markdown lint is reporting an error, and showing a lot of messages for files that are not ours. What's going on, and what do I do?

I'll track those down today.  You can ignore them.

**Review by mairaw — APPROVED** [review]

**Pull request merged**